pkg/apps/strategy/rolling/rolling.go

```diff
@@ -163,12 +162,12 @@ func (s *RollingDeploymentStrategy) Deploy(from *kapi.ReplicationController, to
 	// Execute any pre-hook.
 	if params.Pre != nil {
 		if err := s.hookExecutor.Execute(params.Pre, to, appsapi.PreHookPodSuffix, "pre"); err != nil {
-			return fmt.Errorf("pre hook failed: %s", err)
+			return fmt.Errorf("pre-hook failed: %s", err)
 		}
 	}
 
 	if s.until == "pre" {
-		return strat.NewConditionReachedErr("pre hook succeeded")
+		return strat.NewConditionReachedErr("pre-hook succeeded")
```

Comment:
Isn't this changing our "api" to the user? I know people are grepping these - do we have a good reason to mess with it?

Reply:
there may be some temporarily disabled check for it in tests due to logging issues

Reply:
error messages is not an api

Reply:
that's why the quotes ;)

but this is what oc deploy, rollout and logs will return to the user and they are grepping it

We are allowed to break it, we should just do it intentionally and with some gain.